### ISSUE TITLE
Idea - Bulk add tasks

### ISSUE THREAD

**Issue by jenstornell** [molocLab/kirby-calendar-board#15]
This is an idea. It can be a big one. Now we need to visit every day to set events to them.

In my case with cottages someone has been booking two weeks vacation. That means we need to visit 14 days and setup 14 structure fields for it.

With a bulk kind of input it could be done with just a few clicks.

**How to**

I'm thinking two date fields right below the calendar and after that a structure field that are an exact copy of the day structure field.

On save it has an interval of dates. It also have a structure field with data. With a hook it would be possible to populate all the days in the range.

Feedback?

**Comment by molocLab** [comment]
It's a good argument.

Now calendar board is useful for daily activities or one shot events, but it's unhandy for bookings (from day to day) or in general for repeating events.

I'm looking for the best way to implement this feature.
You solution works, but I fear it will complicate too much calendar board interface.

I'm thinking about this: when you add an event in the structure field inside a day there will be a new field in addition to those already present. This new field will be something like:

`Repeat this event for x day`.

**Comment by jenstornell** [comment]
Yes, that's really brilliant! +1

If I understand it correctly there will be one more field in the blueprint outside the structure field (below it maybe).

Then the field is set to 0 as default. If set to 6 it will add this event to this day and 6 days after that (totally 7 days including this one).

**Sidenote**

I don't know if there also can be relevant to have a direction choice, forward or backward in time? I can live without it, just thinking of completeness.

**Comment by molocLab** [comment]
I actually was thinking to insert the new field inside every event detail.

![download 1](https://cloud.githubusercontent.com/assets/11831276/12810941/a1ac4c52-cb29-11e5-8308-50556c3440e1.png)

**Comment by jenstornell** [comment]
Yes of cource! We probably never want to copy all events from one day. Your screenshot is correct.

**Comment by molocLab** [comment]
First screenshot about "cloning events" feature:

![download_clone](https://cloud.githubusercontent.com/assets/11831276/12919630/56dde760-cf45-11e5-879e-72553101f209.png)

It seems to works, but unfortunately the feature collides with this [panel issue](https://github.com/getkirby/panel/issues/688)

**Comment by jenstornell** [comment]
I don't see from the screenshot what you did to clone them. Too bad it collides with the bug. Is there a way around it?

**Comment by molocLab** [comment]
Distantnative  published a walk around, but it doesn't work for me. (maybe my fault)

**Comment by molocLab** [comment]
Kirby 2.3.1 solved the bug!
